framework/tests/org.jboss.tools.rsp.server.test/src/main/java/org/jboss/tools/rsp/server/model/ServerModelTest.java

```diff
@@ -49,13 +50,14 @@
 public class ServerModelTest {
 	private static final DataLocationSysProp dataLocation = new DataLocationSysProp();
 
-	private static final String BONNET_COLOR_ATTRIBUTE_ID = "Bonnet";
+	private static final String BONNET_COLOR_ATTRIBUTE_ID = "BonnetColor";
 	private static final String HAS_BEARD_ATTRIBUTE_ID = "HasBeard";
+	protected static final String THROUSERS_COLOR_ATTRIBUTE_ID = "throusersColor";
```

Comment:
can you please tell me what "throusers" means? through the users?  I'm a bit confused.

Reply:
"throusers" should be spelled "trousers" ;)